docs/troubleshooting.md

```diff
@@ -290,6 +290,19 @@ There was an issue with the installing of the drivers certificate, so the driver
 [This Guide](guests.md#upgrade-from-citrix-xenserver-client-tools) may help you through the process.
 
 
+### My Windows VM can't connect to a secure internet after installing PV-tools/drivers (E.G SSL errors when trying to access websites, copying files from a pc to the VM ETC.)
```

Comment:
Try to use a shorter title, eg "Windows SSL issues" and then explain more in details if you want after the title with more words :+1: